endaq/ide/files.py

```diff
@@ -91,27 +91,28 @@ def get_doc(name=None, filename=None, url=None, parsed=True, start=0, end=None,
     One and only one must be specified. Attempting to supply more than one
     will generate an error.
 
-    Example usage:
-    ```
-    get_doc("my_recording.ide")
-    get_doc("https://example.com/remote_recording.ide")
-    get_doc(filename="my_recording.ide")
-    get_doc(url="https://example.com/remote_recording.ide")
-    get_doc(filename="my_recording.ide", start="1:23")
-    ```
+    Example usage::
+
+        get_doc("my_recording.ide")
+        get_doc("https://example.com/remote_recording.ide")
+        get_doc(filename="my_recording.ide")
+        get_doc(url="https://example.com/remote_recording.ide")
+        get_doc(filename="my_recording.ide", start="1:23")
 
     The `start` and `end` times, if used, may be specified in several
     ways:
-        * `int`/`float` (Microseconds from the recording start)
-        * `str` (formatted as a time from the recording start, e.g.,
-          `MM:SS`, `HH:MM:SS`, `DDd HH:MM:SS`). More examples:
-            * ``":01"`` or ``":1"`` or ``"1s"`` (1 second)
-            * ``"22:11"`` (22 minutes, 11 seconds)
-            * ``"3:22:11"`` (3 hours, 22 minutes, 11 seconds)
-            * ``"1d 3:22:11"`` (1 day, 3 hours, 22 minutes, 11 seconds)
-        * `datetime.timedelta` or `pandas.Timedelta` (time from the
-          recording start)
-        * `datetime.datetime` (an explicit UTC time)
+
+    * `int`/`float` (Microseconds from the recording start)
+    * `str` (formatted as a time from the recording start, e.g., `MM:SS`,
+      `HH:MM:SS`, `DDd HH:MM:SS`). More examples:
+
```

Comment:
Why the line break here? If it is necessary, why not after the indented section as well?

Reply:
My goal was to make the minimal number of changes required to remove build warnings; the line break there was necessary, and the others weren't.

Having said that, I don't care too much about how these are formatted. So if you'd like to add in matching line breaks for the indented sections, feel free to do so. But I'd ask that you make the changes yourself if you don't mind; I think doing so will avoid a lot of back and forth conversation between us.

Reply:
Yeah, it isn't really important. I'm admittedly a bit particular about formatting. It wasn't meant as a request for change.